content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+    >}}
+
+### Multiple control planes
+
+You can deploy multiple control planes across the mesh; such a configuration is
+known as a **multi-control plane**.
+
+Multiple control planes afford improved performance and availability for meshes
+that span [multiple clusters](#multiple-clusters) across multiple zones and
+regions.
+
+Istio supports control plane deployments at various levels of availability:
+
+- Control plane per region: *lowest availability*
+- Control plane per zone
+- Control plane per cluster *highest availability*
+
+The configuration of each control plane could be identical or different
+depending on the [network](#network-models) and [cluster](#cluster-models)
+models of the system.
+
+{{< image width="50%"
+    link="./multi-control.svg"
+    alt="A service mesh with multiple control planes"
+    title="Multiple control planes"
+    caption="A service mesh with multiple control planes"
+    >}}
+
+### Managed control plane
+
+Many cloud providers offer a control plane, which they manage on behalf of their
+customer.
+
+Typically, managed control planes guarantee some level of performance and
+availability, which you can assume when building your system. Managed control
+planes affords the following benefits:
+
+- Greatly reduce the complexity of user deployments
+- Greatly reduce system administration costs
+- Effectively eliminate the usage costs associated with the control plane:
+    - CPU
+    - Memory
+    - Network
+
+{{< image width="50%"
+    link="./managed-control.svg"
+    alt="A service mesh with a managed control plane"
+    title="Managed control plane"
+    caption="A service mesh with a managed control plane"
+    >}}
+
+## Identity and trust models
+
+When a workload instance is created within a service mesh, Istio assigns the
+workload an identity. An identity is a unique name backed by a key material, for
+example:
+
+- [X.509 certificate](https://en.wikipedia.org/wiki/X.509)
+- [JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)
+
+Each identity Istio assigns is understood throughout the mesh and you can use it
+to enable mutual authentication and enforce policies.
+
+An identity contains a substring identifying the mesh that created it. For
+example in `spiffe://mytrustdomain.com/ns/default/sa/myname` the substring
+identifying the mesh is: `mytrustdomain.com`. We call this substring the {{<
+gloss>}}trust domain{{< /gloss >}} of the mesh. Every Istio mesh has a globally
+unique trust domain used to create identity names.
+
+The Certificate Authority (CA) creates the identities and their key material.
+You can verify the message sender's identity with the public key of the CA that
+created the identity. A **trust bundle** is the set of
+all CA public keys used by an Istio mesh. With a mesh's trust bundle, anyone can
+verify the sender of any message coming from that mesh.
+
+### Trust within a mesh
+
+Within a single Istio mesh, a CA creates identities and ensures that each
```

Comment:
Istio is ensuring this property, not the CA. The CA is just signing the identities. Istio ensures each workload instance has an appropriate certificate representing its own identity, and the trust bundle necessary to recognize all identities in the mesh and in federated meshes.

Reply:
@smawson I'll modify the text accordingly.